tests/unit.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
this file should be called functions.php not unit.php

Reply:
done

Reply:
its still called unit.php?
Have you pushed the changes?